### PR TITLE
Docs - Updated the docs for cron

### PR DESCRIPTION
**Pull request by naveensrinivasan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
N/A

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
 Docs

* **What is the current behavior?** (You can also link to an open issue here)
  #277

* **What is the new behavior (if this is a feature change)?**

   Included a section within the CONTRIBUTING.md about the dailyscore and cron job.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
None

* **Other information**:
None

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,6 +81,21 @@ The personal access token need the following scopes:
 
 1. See the [action files](.github/workflows) to check its tests, and the scripts used on it.
 
+## dailyscore-cronjob
+
+scorecard scans https://github.com/ossf/scorecard/blob/main/cron/projects.txt repositories in GCP k8s and publishes the results in GCS bucket `ossf-scorecards`.
+
+The `cron` definition ../scorecard/cron/cron.sh and the `k8s` for the `cron` is in ../scorecard/k8s/cron.yaml
```
Comment (inferno-chromium) [inline CONTRIBUTING.md:88]: nit: change "../scorecard" to "."

**Review by inferno-chromium — APPROVED** [review]
Can you please add steps to deploy once someone changes cron.sh or cron.yaml file.

File: CONTRIBUTING.md (hunk 1 above)
Comment (naveensrinivasan) [reply CONTRIBUTING.md:88]: Got it. Thanks

**Comment by naveensrinivasan** [comment]
> Can you please add steps to deploy once someone changes cron.sh or cron.yaml file.

Took care of it. Thanks

**Comment by inferno-chromium** [comment]
qq: "The cronjob can be deployed into k8s by running the ./scorecard/k8s/cron.yaml", can you substitute this with cli command for this ?

**Comment by naveensrinivasan** [comment]
> qq: "The cronjob can be deployed into k8s by running the ./scorecard/k8s/cron.yaml", can you substitute this with cli command for this ?

I will do that.

**Pull request merged**